### DISCUSSION TITLE
Is it possible to disable the workspace cycling round left? (workspace_swipe)

### DISCUSSION THREAD

**Discussion by CactiChameleon9** [hyprwm/Hyprland#1052] [Q&A]
Swiping right creates a blank workspace, and I cannot swipe further. However swiping left on workspace 1 brings me to my final non-empty workspace (workspace -1). Is there a way to disable this looping behaviour because it doesn't quite make sense with workspace_swipe?

Thanks. I made a quick diagram below in case my question didn't make sense

```
1, [2], 3, empty
(right)
1, 2, [3], empty
(right)
1, 2, 3, [empty]
(right)
1, 2, 3, [empty]
(left)
1, 2, [3], empty
(left)
1, [2], 3, empty
(left)
[1], 2, 3, empty
(left - unwanted, prefer to stay at 1)
1, 2, [3], empty
(right)
1, 2, 3, [empty]
```

**Comment by vaxerski** [comment]
Nope. You can make an issue for an enhancement

**Reply by CactiChameleon9** [reply]
Will do, thanks!
